src/Bundle/Tests/Routing/RoutesAttributesLoaderTest.php

```diff
@@ -0,0 +1,320 @@
+
+        $routesCollection = $attributesLoader->__invoke();
+
+        $route = $routesCollection->get('show_book_with_serialization_groups');
+        $this->assertNotNull($route);
+        $this->assertEquals('/book/{id}', $route->getPath());
+        $this->assertEquals([
+            '_controller' => 'app.controller.book:showAction',
+            '_sylius' => [
+                'serialization_groups' => ['sylius'],
+            ],
+        ], $route->getDefaults());
+    }
+
+    /**
+     * @test
+     */
+    public function it_generates_routes_from_resource_with_serialization_version(): void
+    {
+        self::bootKernel(['environment' => 'test_with_attributes']);
+
+        $container = static::$container;
+
+        $attributesLoader = $container->get('sylius.routing.loader.routes_attributes');
+
+        $routesCollection = $attributesLoader->__invoke();
+
+        $route = $routesCollection->get('show_book_with_serialization_version');
+        $this->assertNotNull($route);
+        $this->assertEquals('/book/{id}', $route->getPath());
+        $this->assertEquals([
+            '_controller' => 'app.controller.book:showAction',
+            '_sylius' => [
+                'serialization_version' => '1.0',
+            ],
+        ], $route->getDefaults());
+    }
+
+    /**
+     * @test
+     */
+    public function it_generates_routes_from_resource_with_vars(): void
+    {
+        self::bootKernel(['environment' => 'test_with_attributes']);
+
+        $container = static::$container;
+
+        $attributesLoader = $container->get('sylius.routing.loader.routes_attributes');
+
+        $routesCollection = $attributesLoader->__invoke();
+
+        $route = $routesCollection->get('show_book_with_vars');
+        $this->assertNotNull($route);
+        $this->assertEquals('/book/{id}', $route->getPath());
+        $this->assertEquals([
+            '_controller' => 'app.controller.book:showAction',
+            '_sylius' => [
+                'vars' => [
+                    'foo' => 'bar',
+                ],
+            ],
+        ], $route->getDefaults());
+    }
+
+    /**
+     * @test
+     */
+    public function it_generates_routes_from_resource_with_requirements(): void
+    {
+        self::bootKernel(['environment' => 'test_with_attributes']);
+
+        $container = static::$container;
+
+        $attributesLoader = $container->get('sylius.routing.loader.routes_attributes');
+
+        $routesCollection = $attributesLoader->__invoke();
+
+        $route = $routesCollection->get('show_book_with_requirements');
+        $this->assertNotNull($route);
+        $this->assertEquals('/book/{id}', $route->getPath());
+        $this->assertEquals([
+            'id' => '\d+',
+        ], $route->getRequirements());
+    }
+
+    /**
+     * @test
+     */
+    public function it_generates_routes_from_resource_with_priority(): void
+    {
+        if (\PHP_VERSION_ID < 80000 || Kernel::MAJOR_VERSION < 5) {
```

Comment:
I believe `\PHP_VERSION_ID < 80000` check is not needed as SyliusResourceBundle does not support PHP lower than 8.0 anymore

Reply:
done, yes I removed the other ones. I missed this one.